recipes/himalaya/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.30.0"
+
+
+class HimalayaConan(ConanFile):
+    name = "himalaya"
+    license = "GPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Himalaya-Library/Himalaya"
+    description = "C++ library to calculate three-loop corrections to the Higgs boson mass in the MSSM"
+    topics = ("conan", "high-energy", "physics", "hep", "Higgs", "mass", "MSSM")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False], "shared": [True, False]}
+    default_options = {"fPIC": True, "shared": False}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property 
+    def _minimum_compilers_version(self): 
+        return {
+            "Visual Studio": "15", 
+            "gcc": "5", 
+            "clang": "5", 
+            "apple-clang": "5.1", 
+        } 
+
+    def validate(self):
+        if self.settings.compiler.cppstd: 
+            check_min_cppstd(self, "14")
+        minimum_version = self._minimum_compilers_version.get(str(self.settings.compiler), False) 
+        if not minimum_version: 
+            self.output.warn("Himalaya requires C++14. Your compiler is unknown. Assuming it supports C++14.") 
+        elif tools.Version(self.settings.compiler.version) < minimum_version: 
+            raise ConanInvalidConfiguration("Himalaya requires C++14, which your compiler does not support.") 
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("Himalaya does not support {}".format(self.settings.compiler))
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```

Comment:
```suggestion

    def configure(self):
        if self.options.shared:
            del self.options.fPIC

```

Reply:
Many thanks for the proposal!
But tbh, why should fPIC be deleted if `shared` is enabled? Isn't this the wrong way around?

Reply:
Makes no sense disabling fPIC when building shared library, so we disable it. Otherwise, it would affect the package ID, for both values.

Reply:
Ah, now I think I understand. :) You are saying that since Himalaya always builds a shared library, the `fPIC` option can be deleted completely. Is this correct?